services/static.js

```diff
@@ -176,7 +176,7 @@ function encodeCustomMarkerOverlay(o) {
  * A stylable line.
  * @typedef {Object} PathOverlay
  * @property {Object} path
- * @property {Array<[number, number]>} path.coordinates - An array of coordinates
+ * @property {Array<Coordinates>} path.coordinates - An array of coordinates
```

Comment:
Added a small change here to ensure it's clear that these input coordinates should follow the same `[lng, lat]` format.

Reply:
Ah, thanks.